pages/exp.js

```diff
@@ -1,22 +1,22 @@
 import Head from 'next/head'
 import Image from 'next/image'
-import styles from '../styles/Home.module.css'
-
+ 
 export default function ExperiencePage() {
   return (
-    <div className={styles.container}>
+    <div className="mx-auto px-8 flex flex-col items-center justify-center min-h-screen">
```

Comment:
Would is be possible to make this a classname for a div? Instead of having all these attributes like this. You can easily reuse this as well if it was it's own class.

Reply:
i guess you could have that, all I would do is to change the container classname to have those styles since your original styling on padding doesn't help the centering issue